### ISSUE TITLE
bors merge succeeded and PR still open, though target branch was merged into

### ISSUE THREAD

**Issue by AaronFriel** [bors-ng/bors-ng#1605]
Would appreciate confirmation that this is a GitHub issue and not a bors issue.

This PR was merged via `bors` and the target branch is updated with a merge commit, and the commit (singular) in the branch is now part of the master branch:
- https://github.com/pulumi/pulumi/pull/11963

However as you can see (as of 8:35am Pacific, 16:35am UTC), the pull request has not been marked merged.

**Comment by nairb774** [comment]
If it helps with collecting data points, we've got a self-hosted version of Bors running, and are also seeing this behavior of some PRs staying open after the merge has completed. I just started running our instance with debug logging turned on (hat tip to https://github.com/bors-ng/bors-ng/pull/1602) and I'm hoping that might reveal more information (if only nothing is happening).

**Comment by MarinPostma** [comment]
Same here for me. I think github is detecting some non-existing conflicts in the pr and refuses to merge them, even though the PR is actually merged. This is very frustrating.

**Comment by duckinator** [comment]
Another data point in case it helps: encountered this problem with duckinator/bork#289.

**Comment by AaronFriel** [comment]
@duckinator looks like the time frame lines up with ~4 days ago. Looks like there was an internal service outage in GitHub?

Since about 3 days ago, I don't think we've seen this since with many subsequent merges.

**Comment by duckinator** [comment]
Yeah, looks like it roughly lines up with a GitHub service outage. I've not encountered the problem since then, either.
